tests/bluetooth/tester/src/audio/btp_bap_unicast.c

```diff
@@ -626,6 +626,7 @@ static void stream_started(struct bt_bap_stream *stream)
 {
 	struct btp_bap_unicast_stream *u_stream = stream_bap_to_unicast(stream);
 	struct bt_bap_ep_info info;
+	const uint8_t test_data[5] = { 0x01, 0x02, 0x03, 0x04, 0x05 };
```

Comment:
Since we configure the SDU size as a product if Octets_Per_Codec_Frame, Audio_Channel_Allocation and Codec_Frame_Blocks_Per_SDU, I think the size of the test data we send should present that. 

Sending e.g. 5 octets when we've configured the SDU size to be e.g. 40 is invalid behavior, which we arguably should not do. (Ideally the data would also be LC3 encoded, but I don't know if PTS cares about that).

Reply:
I can see your reasoning, however the test specification does not require this. In fact, in some tests (BAP/USR/STR/BV-369-C) it even specifies that Null PDUs can be valid: "The IUT transmits CIS Data PDUs or CIS Null PDUs"

Reply:
What the test specs expect/demand may not always be what PTS expects/demands :D I think there was a test case that failed due to not sending the right amount of octets which is of course sort of a PTS bug, but in some cases it's just easier (or at least faster) to make PTS happy than to wait for a PTS fix, especially since BAP requires this (even if the test spec does not specifically mention it).

Reply:
yeap, better to be compliant even if PTS is not validating it in all cases